ods_ci/utils/scripts/testconfig/generateTestConfigFile.py

```diff
@@ -174,8 +174,10 @@ def get_dashboard_url():
     """
     Get dashboard url for the open data science.
     """
-    cmd = "oc get route -A -o json  | jq '.items[].spec.host' | grep 'dashboard'"
-
+    cmd = (
+        "(oc get route -n opendatahub-operators -o json  | jq '.items[].spec.host' | grep odh-dashboard) || "
```

Comment:
odh to be reviewed

Reply:
on a cluster with the latest odh-nightly installed:

```
$ oc get route -n opendatahub-operators -o json  | jq '.items[].spec.host' | grep odh-dashboard

$ oc get route -n opendatahub -o json  | jq '.items[].spec.host' | grep odh-dashboard 
"odh-dashboard-opendatahub.apps.<redacted>"
```

So `opendatahub` seem to be proper place.

Reply:
perfect, thank you!

Reply:
done

Reply:
note: I just realized that this is from my manual installation, I hope that it doesn't differ when installed via our automation... I think it shouldn't.

Reply:
I will do a run from CI for safety, ty